src/avt/Database/Database/avtGenericDatabase.C

```diff
@@ -3172,21 +3172,6 @@ avtGenericDatabase::GetMesh(const char *meshname, int ts, int domain,
             return NULL;
         }
 
-        //
-        // VTK creates a trivial producer for each data set.  It later does
-        // garbage collection and that takes a long time if we have a lot
-        // of trivial producers.  Make one trivial producer for all
-        // data sets here.
-        //
-        static vtkTrivialProducer* tp = NULL;
-        if(tp == NULL)
-        {
-            tp = vtkTrivialProducer::New();
-            vtkVisItUtility::RegisterStaticVTKObject(tp);
-        }
-        tp->SetOutput(mesh);
-        tp->SetOutput(NULL);
-
```

Comment:
@brugger1 I'm still not sure if this section of code is still needed in VisIt. I tried comparing timings logs from the test suite, but that wasn't very productive. Is there any other empirical study of performance that can be done on VisIt with this code intact and without it?
Or should I put the code back in, but comment out with a caveat it can be uncommented if performance seems poor?

Reply:
The commit message for this mentions that the new pipeline API (probably 8.x based on the 2017 authorship date) doesn't work with this pattern. @berkgeveci or @cjh1?

Reply:
Ah, we discussed this in #3456 already. I never got around to testing with this put back. I just rebased our branch on `develop` as it is today. Once that is in, I can test putting this back.